README.md

```diff
@@ -102,3 +102,52 @@ const InstanceLogger = init(({ name }) => {
 
 const george = InstanceLogger({ name: 'George' }); // Created new instance: George
 ```
+
+
+### overrides()
```

Comment:
Best choice! I'll use that name. Expect more commits to this PR. And thanks guys!